src/clear.js

```diff
@@ -0,0 +1,23 @@
+export const clearing = (e) => {
+  const click = e.target.closest('.check');
+  if (!click) return;
+
+  const dataT = parseInt(click.getAttribute('data-set'), 10);
```

Comment:
- [ ] In clear.js: I would suggest you name your variables in a way that they reveal the intention behind it.

Reply:
Done! Replaced dataT for localCheck 🔧